integration-tests/plugins/__tests__/regions/admin/update-region.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IRegionModuleService } from "@medusajs/types"
+import path from "path"
+import { startBootstrapApp } from "../../../../environment-helpers/bootstrap-app"
+import { useApi } from "../../../../environment-helpers/use-api"
+import { getContainer } from "../../../../environment-helpers/use-container"
+import { initDb, useDb } from "../../../../environment-helpers/use-db"
+import adminSeeder from "../../../../helpers/admin-seeder"
+
+jest.setTimeout(50000)
+
+const env = { MEDUSA_FF_MEDUSA_V2: true }
+const adminHeaders = {
+  headers: { "x-medusa-access-token": "test_token" },
+}
+
+describe("POST /admin/regions/:id", () => {
```

Comment:
I think having a single `regions/admin/regions.spec.ts` file will reduce all the boilerplate around setting up the before/after hooks, and it also makes sense for CRUD operations to be next to each other. Since these are integration tests, you can have a more detailed flow in a single test where you:
1. Create a region
2. You update it
3. You delete it

This tests the entire lifecycle of the region without duplicating a lot of code.